### PR TITLE
Couple of nitpick fixes

### PR DESCRIPTION
**Pull request by AndreMiras**
## What was wrong?

Nothing wrong simply a couple of styling/typo fixes.

## How was it fixed?

Made the non-functional changes.

#### Cute Animal Picture

image

### CODE REVIEW

File: eth_account/account.py [hunk 1]
```diff
@@ -55,7 +55,7 @@
 )
 
 
-class Account(object):
+class Account:
```
Comment (AndreMiras) [inline eth_account/account.py]: In Python3 object is inherited by default

File: eth_account/_utils/validation.py [hunk 2]
```diff
@@ -5,9 +5,4 @@
 
 
 def is_valid_address(value):
-    if is_binary_address(value):
-        return True
-    elif is_checksum_address(value):
-        return True
-    else:
-        return False
+    return is_binary_address(value) or is_checksum_address(value)
```
Comment (AndreMiras) [inline eth_account/_utils/validation.py]: less if often more

**Comment by pacrob** [comment]
Thanks for the PR, and sorry for the delay in moving on it. Something went funky with the CI, so I opened a new PR #179 but was able to use the original commits.

**Comment by fselmo** [comment]
closed by #179

**Pull request closed without merging**